hcatalog/core/src/main/java/org/apache/hive/hcatalog/cli/SemanticAnalysis/HCatSemanticAnalyzer.java

```diff
@@ -21,10 +21,10 @@
 import org.apache.hadoop.hive.metastore.api.Database;
 import org.apache.hadoop.hive.ql.ddl.DDLDesc;
 import org.apache.hadoop.hive.ql.ddl.DDLWork;
-import org.apache.hadoop.hive.ql.ddl.database.DescDatabaseDesc;
-import org.apache.hadoop.hive.ql.ddl.database.DropDatabaseDesc;
-import org.apache.hadoop.hive.ql.ddl.database.ShowDatabasesDesc;
-import org.apache.hadoop.hive.ql.ddl.database.SwitchDatabaseDesc;
+import org.apache.hadoop.hive.ql.ddl.database.desc.DescDatabaseDesc;
+import org.apache.hadoop.hive.ql.ddl.database.drop.DropDatabaseDesc;
+import org.apache.hadoop.hive.ql.ddl.database.show.ShowDatabasesDesc;
+import org.apache.hadoop.hive.ql.ddl.database.switchdb.SwitchDatabaseDesc;
```

Comment:
"switch" is a reserved word in java, and thus it can not be used even as a segment of a package name :)

Bt probably it could be modified to "use". Do you agree?

Reply:
I did not realize about `switch`... `use` is fine.

Reply:
Switched switch to use.